utils/bazel/llvm-project-overlay/libc/test/src/__support/BUILD.bazel

```diff
@@ -0,0 +1,90 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tests for LLVM libc __support functions.
+
+load("//libc/test:libc_test_rules.bzl", "libc_test")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
[nit] Do we need public here ?

Reply:
Let's do this in a separate patch if you don't mind since there are several other files to fix.